venue.html

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Venue
+permalink: /venue/
+---
+<div class="col-md-10 col-md-push-1">
+  <h1>Venue</h1>
+  <p><a href="http://www.cit.edu" target="_blank"><img src="{{ "/images/CIT-University.png" | relative_url }}" alt="CIT University logo" style="max-width: 200px;"/></a></p>
```

Comment:
I think it would be better to keep the styling in SASS. "max-width: 200px;" inside a SASS file. What do you think?